code/src/settings.c

```diff
@@ -94,6 +96,24 @@ void FairyPickupHealAmount(void) {
     }
 }
 
+u32 Settings_GetQuickTextOption() {
+    return gSettingsContext.quickText;
+}
+
+u32 Settings_IsTurboText() {
+    return (gSettingsContext.quickText >= QUICKTEXT_TURBO && buttonCheck(BUTTON_B));
```

Comment:
I think instead of using `buttonCheck` you might just want to check `(rInputCtx.cur & BUTTON_B)`. Haven't tested, but i think that will fix the issue of being able to hold other buttons, and i think `buttonCheck` is meant to stay internal to `input.c` anyway

Reply:
Done. As you predicted, this fixed that issue. Thank you